src/vision.cs

```diff
@@ -2302,5 +2607,815 @@ interface VNRequestProgressProviding {
+
+		[Wrap ("GetRecognizedPoints (bodyLandmarkRegion.GetConstant ()!, out error)")]
+		[return: NullAllowed]
+		NSDictionary<NSString, VNRecognizedPoint> GetRecognizedPoints (VNHumanBodyPoseObservationJointsGroupName bodyLandmarkRegion, out NSError error);
+
+		[Wrap ("GetRecognizedPoints (handLandmarkRegion.GetConstant ()!, out error)")]
+		[return: NullAllowed]
+		NSDictionary<NSString, VNRecognizedPoint> GetRecognizedPoints (VNHumanHandPoseObservationJointsGroupName handLandmarkRegion, out NSError error);
+
+		[Export ("keypointsMultiArrayAndReturnError:")]
+		[return: NullAllowed]
+		MLMultiArray GetKeypoints (out NSError error);
+	}
+
+	[TV (14,0), Mac (11,0), iOS (14,0)]
+	[BaseType (typeof (VNImageBasedRequest))]
+	[DisableDefaultCtor]
+	interface VNStatefulRequest {
+
+		[Export ("initWithFrameAnalysisSpacing:completionHandler:")]
+		IntPtr Constructor (CMTime frameAnalysisSpacing, [NullAllowed] VNRequestCompletionHandler completionHandler);
+
+		[Export ("minimumLatencyFrameCount")]
+		nint MinimumLatencyFrameCount { get; }
+
+		[Export ("frameAnalysisSpacing")]
+		CMTime FrameAnalysisSpacing { get; }
+
+		// We must inline the following 5 properties
+		// ('Revision', 'WeakSupportedRevisions', 'SupportedRevisions', 'DefaultRevision' and 'CurrentRevision')
+		// into subclasses so the correct class_ptr is used for the static members and the right enum type is also used.
+
+		[Export ("revision")]
+		VNStatefulRequestRevision Revision { get; set; }
+
+		[Static]
+		[Export ("supportedRevisions", ArgumentSemantic.Copy)]
+		NSIndexSet WeakSupportedRevisions { get; }
+
+		[Static]
+		[Wrap ("GetSupportedVersions<VNStatefulRequestRevision> (WeakSupportedRevisions)")]
+		VNStatefulRequestRevision [] SupportedRevisions { get; }
+
+		[Static]
+		[Export ("defaultRevision")]
+		VNStatefulRequestRevision DefaultRevision { get; }
+
+		[Static]
+		[Export ("currentRevision")]
+		VNStatefulRequestRevision CurrentRevision { get; }
+	}
+
+	[Deprecated (PlatformName.MacOSX, 11, 0, message: "Use 'VNVideoProcessorRequestProcessingOptions' instead.")]
+	[Deprecated (PlatformName.iOS, 14, 0, message: "Use 'VNVideoProcessorRequestProcessingOptions' instead.")]
+	[Deprecated (PlatformName.TvOS, 14, 0, message: "Use 'VNVideoProcessorRequestProcessingOptions' instead.")]
+	[TV (11,0), Mac (10,13), iOS (11,0)]
+	[Internal]
+	[Static]
+	interface VNVideoProcessingOptionKeys {
+		[Field ("VNVideoProcessingOptionFrameCadence")]
+		NSString FrameCadenceKey { get; }
+
+		[Field ("VNVideoProcessingOptionTimeInterval")]
+		NSString TimeIntervalKey { get; }
+	}
+
+	[Deprecated (PlatformName.MacOSX, 11, 0, message: "Use 'VNVideoProcessorRequestProcessingOptions' instead.")]
+	[Deprecated (PlatformName.iOS, 14, 0, message: "Use 'VNVideoProcessorRequestProcessingOptions' instead.")]
+	[Deprecated (PlatformName.TvOS, 14, 0, message: "Use 'VNVideoProcessorRequestProcessingOptions' instead.")]
+	[TV (11,0), Mac (10,13), iOS (11,0)]
```

Comment:
same

Reply:
This got replaced at a later beta heh so it was a left over of previous iteration :P nice catch